src/main/java/com/google/sps/servlets/AddTA.java

```diff
@@ -30,30 +37,63 @@ public void init(ServletConfig config) throws ServletException {
     }
   }
 
-  // Add a TA to the datastore
+  // Add a TA to the class datastore
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
+    int retries = 10;
+
     try {
```

Comment:
try { while(true) { try { ... }}} is a confusing construction -- i'd either simplify or restructure with some helper methods